Packs/PrismaCloud/Integrations/RedLock/RedLock.py

```diff
@@ -525,17 +525,24 @@ def fetch_incidents():
     Retrieve new incidents periodically based on pre-defined instance parameters
     """
     now = int((datetime.utcnow() - datetime.utcfromtimestamp(0)).total_seconds() * 1000)
-    last_run = demisto.getLastRun().get('time')
-    if not last_run:  # first time fetch
-        last_run = now - parse_date_range(demisto.params().get('fetch_time', '3 days').strip(), to_timestamp=True)[0]
-
-    payload = {'timeRange': {
-        'type': 'absolute',
-        'value': {
-            'startTime': last_run,
-            'endTime': now
-        }
-    }, 'filters': [{'name': 'alert.status', 'operator': '=', 'value': 'open'}]}
+    last_run = demisto.getLastRun()
+    last_incident_time = last_run.get('time')
+    last_fetches = last_run.get('fetches', [])
+    if not last_incident_time:  # first time fetch
+        last_incident_time = parse_date_range(demisto.params().get('fetch_time', '3 days').strip(), to_timestamp=True)[0]
+
+    payload = {
+        'timeRange': {
+            'type': 'absolute',
+            'value': {
+                'startTime': last_incident_time,
+                'endTime': now
+            }
+        },
+        'filters':
+            [{'name': 'alert.status', 'operator': '=', 'value': 'open'}],
+        'sortBy': ['alertTime:asc'],
+        'limit': demisto.getParam('limit')}
```

Comment:
Please add a default value in case this is empty, and use max function with the default value.

Make sure to add to `test-module` a check that the user didn't provide a value greater than the default value.